core/flyout_base.js

```diff
@@ -524,10 +514,10 @@ Blockly.Flyout.prototype.show = function(flyoutDef) {
 /**
  * Create the contents array and gaps array necessary to create the layout for
  * the flyout.
- * @param {Array.<Blockly.utils.toolbox.Toolbox>} parsedContent The array
- *   of objects to show in the flyout.
+ * @param {!Array<Blockly.utils.toolbox.FlyoutItem>} parsedContent The array
+ *     of objects to show in the flyout.
  * @return {{contents:Array.<Object>, gaps:Array.<number>}} The list of contents
```

Comment:
No :/ This is legacy code and I am a bit hesitant to add one since the object is around for such a short amount of time and I feel like it will confuse the flyout types we already have. However, if you think it is important happy to add a type.